package.json

```diff
@@ -56,7 +56,10 @@
     "package": "npm run build:fast && npm run build:standalone && npm run build:installer",
     "release": "ts-node --transpile-only automation/run.ts release",
     "pretest": "npm run build",
-    "test": "mocha --timeout 6000 -r ts-node/register/transpile-only \"tests/**/*.spec.ts\"",
+    "test": "npm run test:source && npm run test:standalone",
+    "test:source": "mocha --timeout 6000 -r ts-node/register/transpile-only \"tests/**/*.spec.ts\"",
+    "test:standalone": "npm run build:standalone && cross-env BALENA_CLI_TEST_TYPE=standalone npm run test:source",
+    "test:standalone:fast": "cross-env BALENA_CLI_TEST_TYPE=standalone npm run test:source",
```

Comment:
I should highlight these changes for reviewing purposes. The are related to enabling the tests to run with the standalone executable. Btw, see comments at the top of `tests/proxy-server.ts` for an overview of the tests involving a proxy server.

Reply:
Nice!